### ISSUE TITLE
Update how to start DYOM guide

### ISSUE THREAD

**Issue by moziliar** [nusdistsys/NUS_DYOM#2]
Perhaps it should be prioritised over other sections? The arguments are:

* The process is actually simple and clear cut, yet less known to students who are less driven (aka. it is fine to not have) and therefore setting up a mental barrier for them
* The supervisor might not know what his/her responsibility and workload are in it, making them less willing to supervise in the first place (and hence making the process even harder)

We should probably gather advice from both student hosts and the supervisors so as to clear both of the above.

**Comment by J0** [comment]
Yes, this makes sense. There's definitely a lot of overhead to setting up a DYOM if you don't already know what you're looking for. Let's pick up this discussion post exams and maybe rope the cat theory and algo dyom in.

**Comment by J0** [comment]
Ok bumping this thread, shall we compile a list of instructions?
